Rakefile

```diff
@@ -27,143 +27,135 @@ namespace :test do
+         ## beaker/host.rb is updated to work with passwords.
+         # '--tests', beaker_test_base_dir,
+         # '--load-path', load_path_option,
+         '--tests', 'acceptance/tests/',
+         '--log-level', 'debug',
+         '--debug')
     end
   end
 end
 
 # namespace-named default tasks.
 # these are the default tasks invoked when only the namespace is referenced.
 # they're needed because `task :default` in those blocks doesn't work as expected.
-task 'test:spec' => 'test:spec:run'
-task 'test:acceptance' => 'test:acceptance:quick'
+task 'test:spec': %i[test:spec:run]
+task 'test:acceptance': %i[test:acceptance:quick]
 
 # global defaults
-task :lint => %i[lint:rubocop]
-task :test => 'test:spec'
-task :default => :test
+task lint: %i[lint:rubocop]
+task test: %i[test:spec]
+task default: %i[test]
 
 ###########################################################
 #
 #   Documentation Tasks
 #
 ###########################################################
-DOCS_DAEMON = "yard server --reload --daemon --server thin"
+DOCS_DAEMON = 'yard server --reload --daemon --server thin'
 FOREGROUND_SERVER = 'bundle exec yard server --reload --verbose --server thin lib/beaker'
 
-def running?( cmdline )
+def running?(cmdline)
   ps = `ps -ef`
-  found = ps.lines.grep( /#{Regexp.quote( cmdline )}/ )
-  if found.length > 1
-    raise StandardError, "Found multiple YARD Servers. Don't know what to do."
-  end
+  found = ps.lines.grep(/#{Regexp.quote(cmdline)}/)
+  raise StandardError, "Found multiple YARD Servers. Don't know what to do." if found.length > 1
 
-  yes = found.empty? ? false : true
-  return yes, found.first
+  [!found.empty?, found.first]
 end
 
-def pid_from( output )
-  output.squeeze(' ').strip.split(' ')[1]
+def pid_from(output)
+  output.squeeze(' ').strip.split[1]
 end
 
 desc 'Start the documentation server in the foreground'
-task :docs => 'docs:clear' do
-  original_dir = Dir.pwd
-  Dir.chdir( File.expand_path(File.dirname(__FILE__)) )
-  sh FOREGROUND_SERVER
-  Dir.chdir( original_dir )
+task docs: %i[docs:clear] do
+  Dir.chdir(__dir__) do
+    sh FOREGROUND_SERVER
+  end
 end
 
 namespace :docs do
   desc 'Clear the generated documentation cache'
   task :clear do
-    original_dir = Dir.pwd
-    Dir.chdir( File.expand_path(File.dirname(__FILE__)) )
-    sh 'rm -rf docs'
-    Dir.chdir( original_dir )
+    Dir.chdir(__dir__) do
+      sh 'rm -rf docs'
+    end
   end
 
   desc 'Generate static documentation'
-  task :gen => 'docs:clear' do
-    original_dir = Dir.pwd
-    Dir.chdir( File.expand_path(File.dirname(__FILE__)) )
-    output = `bundle exec yard doc`
-    puts output
-    if output =~ /\[warn\]|\[error\]/
-      fail "Errors/Warnings during yard documentation generation"
+  task gen: %i[docs:clear] do
+    Dir.chdir(__dir__) do
+      output = `bundle exec yard doc`
+      puts output
+      raise 'Errors/Warnings during yard documentation generation' if /\[warn\]|\[error\]/.match?(output)
```

Comment:
I'm against `Style/IfUnlessModifier`. The problem IMHO is that it tries to fill the line to a max length, but IMHO it's too wide to understand the control flow.  So I very much prefer the old version where it was a traditional if statement.